### PR TITLE
add all groups to RDF

### PR DESCRIPTION
**Pull request by stefina**

### CODE REVIEW

File: ckanext/stadtzhtheme/dcat/profiles.py [hunk 1]
```diff
@@ -203,9 +203,16 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
...
         groups = self._get_dataset_value(dataset_dict, 'groups')
         try:
-            group_id = groups[0].get('id')
-            theme_ids = self._themes(group_id)
-            for theme_id in theme_ids:
+            theme_ids = []
+            for group in groups:
+                group_id = group.get('id')
+                if self._themes(group_id)[0]:
+                    theme_ids.append(str(self._themes(group_id)[0]))
```
Comment (metaodi) [inline ckanext/stadtzhtheme/dcat/profiles.py]: Please append all themes, not just the first one

Comment (metaodi) [inline ckanext/stadtzhtheme/dcat/profiles.py]: You can write this whole block as a list comprehension and use itertools.chain.from_iterable to flatten the list:

```python
import itertools
theme_ids = set(itertools.chain.from_iterable([self._themes(group.get('id')) for group in 
groups]))
```

File: ckanext/stadtzhtheme/dcat/profiles.py [hunk 2]
```diff
@@ -279,8 +286,6 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
 
             #  Lists
             items = [
-                ('theme', DCAT.theme, None),
-                ('language', DCT.language, None),
```
Comment (metaodi) [inline ckanext/stadtzhtheme/dcat/profiles.py]: Why is languages removed here?

Comment (stefina) [inline ckanext/stadtzhtheme/dcat/profiles.py]: I realized that DCT.language was already set in line 135. Setting DCAT.theme was also a duplicate. We set language like so: g.add((dataset_ref, DCT.language, Literal(ckan_locale_default)))

**Comment by stefina** [comment]
This one I would merge first. PR https://github.com/opendatazurich/ckanext-stadtzh-theme/pull/58 also contains these changes, as it based from this branch.

**Pull request merged**